src/clusterfuzz/_internal/bot/fuzzers/afl/launcher.py

```diff
@@ -68,6 +69,9 @@
 # This will no longer be needed when we migrate to the engine interface.
 POSTPROCESSING_TIMEOUT = 30
 
+# Window of time for afl to exit gracefully before we kill it.
+AFL_CLEAN_EXIT_TIME = 10.0
```

Comment:
so afl-cmin is run for 10 (seconds)? and is terminated afterwards?
afl-cmin is designed not to hang, so terminating it when not complete will result in an incomplete minimizing and the output must then be wiped ...

Reply:
Not quite. This is just some extra time beyond the max time ClusterFuzz gives for minimization (i'm not sure what that is (@oliverchang ?) but it's definitely more than 10 seconds). See https://github.com/google/clusterfuzz/pull/2653/files/17887a53f75061ee2b634fc7bf1d16b68178cc3f#diff-92257cb520950616d0435af83ad777608e51235269b56b54081f54bcf68de2e1R1503